templates/_kitchen-sink.twig

```diff
@@ -0,0 +1,234 @@
+{% if not devMode %}
+  {% requireLogin %}
+  {% if not currentUser.admin %}
+    {% exit 403 %}
+  {% endif %}
+{% endif %}
+
+
+
+{% extends '_layouts/base' %}
+{% set doNotCache = true %}
+
+{% set loremTitle = 'Aperiam, culpa numquam!' %}
+{% set loremPara = 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aperiam, culpa numquam! Numquam corrupti rerum, nihil explicabo voluptatibus aliquam dolorum recusandae pariatur in, porro asperiores dolores similique exercitationem necessitatibus. Ratione, sint.' %}
+
+{% set layoutStyles = [
```

Comment:
Would be good to have at least one example here, maybe as a comment? The other arrays have placeholders so one can refer to those, but the layout one is empty.